examples/dt/uni01alpha/README.md

```diff
@@ -0,0 +1,259 @@
+
+```YAML
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+    service: cinder
+  name: 91-master-cinder-enable-multipathd
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/multipath.conf
+          overwrite: false
+          # Mode must be decimal, this is 0600
+          mode: 384
+          user:
+            name: root
+          group:
+            name: root
+          contents:
+            # Source can be a http, https, tftp, s3, gs, or data as defined in rfc2397.
+            # This is the rfc2397 text/plain string format
+            source: data:,defaults%20%7B%0A%20%20user_friendly_names%20no%0A%20%20recheck_wwid%20yes%0A%20%20skip_kpartx%20yes%0A%20%20find_multipaths%20yes%0A%7D%0A%0Ablacklist%20%7B%0A%7D
+    systemd:
+      units:
+      - enabled: true
+        name: multipathd.service
+```
+
+The plain text contents of the multipath configuration file is
+
+```conf
+defaults {
+  user_friendly_names    no
+  recheck_wwid    yes
+  skip_kpartx    yes
+  find_multipaths  yes
+}
+blacklist {
+}
+```
+
+#### Cinder backend - LVM
+
+It is assumed that worker nodes or the master nodes have extra disks and there
+exists a logical volume with the name *cinder_volumes*. If not, a
+`MachineConfig` like the one below must be applied.
+
+```YAML
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 92-create-logical-volume
+  labels:
+    machineconfiguration.openshift.io/role: master
+    service: cinder
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+        - device: "/dev/sdb"
+          wipeTable: true
+        - device: "/dev/sdc"
+          wipeTable: true
+      files:
+        - path: /usr/local/bin/lv-cinder-volumes.sh
+          overwrite: false
+          mode: 493
+          user:
+            name: root
+          group:
+            name: root
+          contents:
+            source: data:text/plain;charset=utf-8;base64,{{ _replaced_ }}
+    systemd:
+      units:
+        - name: lv-cinder-volumes.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Create logical volume with name cinder-volumes.
+            After=var.mount systemd-udev-settle.service
+
+            [Service]
+            Type=oneshot
+            ExecStart=/usr/local/bin/lv-cinder-volumes.sh
+            RemainAfterExit=yes
+
+            [Install]
+            WantedBy=multi-user.target
+```
+
+*{{ _replaced_ }}* should be replaced with the base64 encoding of the below script.
+
+```bash
+#! /usr/bin/bash
+set -euo pipefail
+
+if [[ $(lvdisplay /dev/cinder_volumes_vg/cinder-volumes) ]]; then
+    echo "cinder-volumes vg exists."
+    exit 0
+fi
+
+disks=$(lsblk -o NAME,TYPE | awk '{ if ($2 == "disk" && $1 != "sda") print "/dev/"$1}')
+disk_str=''
+
+for disk in ${disks}; do
```

Comment:
This is fine, but overkill. CI needs ~20 GB space, and traditionally we used a loopback device on a locally created file. Switching to a dedicated disk (/dev/sdb) is a good idea, but I don't see a need to support multiple disks. I also don't know if this means you'll need to allocate extra disks to every OCP node even though only one of them will host cinder's VG. See my comment below.

Reply:
Thank you.

Yes, we are adding two extra disks as part of the deployment. It is an overkill but we wanted to ensure all requirements would be covered hence the generalization.